filehandler/handle_audio.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import os
+
+
```

Comment:
Why is this spacing so big? They are all import statements, right?
I think it should be without spacing like in this:
https://www.python.org/dev/peps/pep-0008/#imports